pyxrf/model/utils.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import scipy
+
+# =================================================================================
+#  The following set of functions are separated from the rest of the program
+#  and prepared to be moved to scikit-beam (skbeam.core.fitting.xrf_model)
```

Comment:
This can (optionally) be made as a module docstring in the very beginning of the file.